crates/strategy/zksync/src/cheatcode/runner/cheatcode_handlers.rs

```diff
@@ -332,6 +333,23 @@ impl ZksyncCheatcodeInspectorStrategyRunner {
                 // We need to return the fork ID.
                 Ok(encoded_fork_id)
             }
+            t if using_zk_vm && is::<broadcastRawTransactionCall>(t) => {
+                let broadcastRawTransactionCall { data } =
```

Comment:
great! What's stopping you from doing the parsing here and calling the original `transaction_from_tx` with the "right" `TransactionRequest`?